akka-persistence/src/main/scala/akka/persistence/fsm/PersistentFSM.scala

```diff
@@ -482,6 +482,9 @@ abstract class AbstractPersistentFSM[S <: FSMState, D, E]
    * Used for identifying domain events during recovery
    */
   def domainEventClass: Class[E]
+
+  // workaround, possibly for https://github.com/scala/bug/issues/11512
+  override def receive: Receive = super.receive
```

Comment:
still needed?

Reply:
Yes - I tried removing it and that didn't work. It seems like rather a corner case, the workaround is simple enough, and it's in PersistentFSM - so I figured it would not be worth getting to the bottom of it in this case.

Reply:
good